Resources/config/services.yml

```diff
@@ -64,3 +69,10 @@ services:
             - "%kernel.bundles%"
         tags:
             - { name: "support_tools.system_info.collector", identifier: "symfony_kernel" }
+
+    # SystemInfoOutputFormats
+
+    support_tools.system_info.output_format.json:
+        class: "%support_tools.system_info.output_format.json.class%"
+        tags:
+            - { name: "support_tools.system_info.output_format", identifier: "json" }
```

Comment:
The [convention is to use ´alias´](http://symfony.com/doc/current/service_container/tags.html#adding-additional-attributes-on-tags), it's not a strong convention so I'll leave it up to @bdunogier if we should follow upstream or stay consistent with other tags here.